google/ggrc-core
# GGRC-7618 Add warn message when user tries to add incorrect folder via import

# Issue description

Add warn message when user tries to add folder to the object via import using GDrive link 

# Steps to test the changes

1. Export any object to Google Sheet, e.g. Objective.
2. Add incorrect value( like 'blahblah3') into ''GDrive Folder id" cell.  
3. Import Objective.
4. Warning message should be rised.


# Solution description

1. Column handler added  for 'folder' object in src/ggrc/cosrc/ggrc/converters/column_handlers.py.
2. New handler (GDriveFolderColumnHandler) added for processing import "folder" object in src/ggrc/converters/handlers/handlers.py.
3. All  changes covered by integration tests.


# Sanity checklist




# PR Review checklist
